src/main/java/org/skriptlang/skript/bukkit/toolcomponent/elements/ExprNewToolComp.java

```diff
@@ -0,0 +1,56 @@
+package org.skriptlang.skript.bukkit.toolcomponent.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.components.ToolComponent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("New Tool Component")
+@Description("Gets a blank tool component.")
+@Examples({
+	"set {_component} to a new blank tool component",
+	"set the tool component of {_item} to {_component}"
+})
+@RequiredPlugins("Minecraft 1.20.6+")
+@Since("INSERT VERSION")
+public class ExprNewToolComp extends SimpleExpression<ToolComponent> {
+
+	static {
+		Skript.registerExpression(ExprNewToolComp.class, ToolComponent.class, ExpressionType.SIMPLE,
```

Comment:
I'm not sure how I feel about a method to create a tool component out of nothing. I'm personally against this I'll wait to see what other members say tho.